pkg/orchestrator/config/config.go

```diff
@@ -63,12 +62,8 @@ func key(pieces ...string) string {
 }
 
 // LoadYamlConfig load orchestrator-specific configuration
-func (oc *OrchestratorConfig) LoadYamlConfig(path string) error {
-	// Resolve any secrets
-	if err := config.ResolveSecrets(config.Datadog, filepath.Base(path)); err != nil {
-		return err
-	}
-
+// at this point secrets should already be resolved by the core/process/cluster agent
+func (oc *OrchestratorConfig) LoadYamlConfig() error {
```

Comment:
nit: you could rename it `LoadConfig` since it doesn't load any yaml file anymore?